src/components/common/MenuHeader.js

```diff
@@ -0,0 +1,22 @@
+import React, { PureComponent } from 'react';
+import { string, func } from 'prop-types';
+
+export default class MenuHeader extends PureComponent {
```

Comment:
This component should be stateless since you are not retaining an internal state and you are not using the react lifecycle methods.